app/retail/emails.py

```diff
@@ -771,6 +793,7 @@ def render_gdpr_update(to_email):
         'terms_of_use_link': 'https://gitcoin.co/legal/terms',
         'privacy_policy_link': 'https://gitcoin.co/legal/privacy',
         'cookie_policy_link': 'https://gitcoin.co/legal/cookie',
+        'email_type': 'roundup',
```

Comment:
This one should definitely be its own type - it may be such that we don't even let people unsubscribe from this one unless they delete their account - what do you think @owocki ?

Reply:
Waiting for your response here as well @owocki

Reply:
yes pls make it its own type.  you can do this by adding it in `retail/emails.py` by adding it to `ALL_EMAILS`